test/cli/cli-config-utils.spec.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const path = require('path');
+const { execFile } = require('child_process');
+const { expect } = require('chai');
+
+/**
+ * Execute _justexit.js as a child process, return a promise for its stdout/stderr
+ *
+ * @param {...string} args
+ * @returns {Promise<{error: import('child_process').ExecException|null, stdout: string, stderr: string}>}
+ */
+function exec(...args) {
+  return new Promise((resolve) => {
+    execFile(
+      'node',
+      [path.resolve(__dirname, '_justexit.js'), ...args],
+      (error, stdout, stderr) => {
+        resolve({ error, stdout, stderr });
+      },
+      { timeout: 1000, cwd: __dirname, env: {} }
+    );
+  });
+}
```

Comment:
cli-testlab doesn't allow for negative assertions on stderr output; but also, with the current state of JS and node, it's pretty ergonomic to just avoid the dependency entirely. Here, instead of deferring the assertions to a library that would have to be updated and published to support more things, we can just get the data directly and perform assertions using whatever assertion library _we_ are using